examples/media/index.html

```diff
@@ -1,63 +1,221 @@
-          </li>
-          <li class="separator"></li>
-        </ul>
+    <main class="container">
+      <div class="container-filters" data-layout="desktop">
+        <div class="container-filters-left">
+          <button data-action="toggle-filters" class="container-filters-button">
+            <svg viewBow="0 0 16 13">
+              <path
+                d="M2.91 12V7.72m0-2.44V1M8 12V6.5m0-2.44V1m5.09 11V8.94m0-2.44V1M1 7.72h3.82m1.27-3.66h3.82m1.27 4.88H15"
+                fill="none"
+                fill-rule="evenodd"
+                stroke="#7C56DD"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
+              />
+            </svg>
 
-        <div id="genres"></div>
-        <div id="ratings"></div>
+            <h2 class="container-filters-title">
+              Filter
+            </h2>
+          </button>
 
-        <div class="thank-you">
-          Data courtesy of <a href="https://www.imdb.com/">imdb.com</a>
+          <div data-widget="stats" class="container-filters-filter"></div>
+          <div
+            data-widget="clear-filters-desktop"
+            class="container-filters-filter"
+          ></div>
+        </div>
+      </div>
+
+      <div
+        id="filters"
+        class="container-filters-panel container-filters-panel--hidden"
+      >
+        <div
+          data-layout="desktop"
+          class="container-filters-panel-column-wrapper"
+        >
+          <div
+            data-widget="categories-desktop"
+            class="container-filters-panel-column"
+          ></div>
+        </div>
+        <div data-layout="mobile" class="container-filters-panel-mobile">
+          <div class="container-filters-header">
+            <h2 class="container-filters-title">
+              Filters
+            </h2>
+            <div data-widget="clear-filters-mobile"></div>
+          </div>
+          <div data-widget="categories-mobile"></div>
+          <div class="container-filters-footer">
+            <button class="see-results-button">
+              <div data-widget="see-results-button"></div>
+            </button>
+          </div>
         </div>
-      </aside>
-
-      <article>
-        <div id="stats" class="text-right text-muted"></div>
-        <hr />
-        <div id="hits"></div>
-        <div id="pagination" class="text-center"></div>
-      </article>
-    </section>
-
-    <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.includes"></script>
-    <script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.0.0-beta.13/dist/algoliasearch-lite.umd.min.js"></script>
-    <script src="instantsearch.production.min.js"></script>
-    <script src="search.js"></script>
+      </div>
+
+      <section class="container-results">
+        <div data-widget="hits"></div>
+      </section>
+    </main>
+
+    <aside class="back-to-top">
+      <a href="#" title="Back to top">
```

Comment:
as the href is #, it removes the search parameters, not sure what can be used to keep search though

Reply:
I can try to prevent its default behavior, so it won't follow through the route change with the #.

Reply:
I changed the "back to top" behavior in 6561751.